src/sagemaker_containers/_files.py

```diff
@@ -47,22 +50,23 @@ def write_failure_file(failure_msg):  # type: (str) -> None
 
 
 @contextlib.contextmanager
-def tmpdir(suffix="", prefix="tmp", dir=None):  # type: (str, str, str) -> None
-    """Create a temporary directory with a context manager. The file is deleted when the context exits.
+def tmpdir(suffix="", prefix="tmp", directory=None):  # type: (str, str, str) -> None
+    """Create a temporary directory with a context manager. The file is deleted when the
+    context exits.
 
     The prefix, suffix, and dir arguments are the same as for mkstemp().
 
     Args:
-        suffix (str):  If suffix is specified, the file name will end with that suffix, otherwise there will be no
-                        suffix.
-        prefix (str):  If prefix is specified, the file name will begin with that prefix; otherwise,
-                        a default prefix is used.
-        dir (str):  If dir is specified, the file will be created in that directory; otherwise, a default directory is
-                        used.
+        suffix (str):  If suffix is specified, the file name will end with that suffix,
+                       otherwise there will be no suffix.
+        prefix (str):  If prefix is specified, the file name will begin with that prefix;
+                       otherwise, a default prefix is used.
+        directory (str):  If directory is specified, the file will be created in that directory;
```

Comment:
Love it! Treat every variable like you need to be able to process it at 2am with bad lighting and a migraine =)